gcsio/src/main/java/com/google/cloud/hadoop/gcsio/authorization/AuthorizationHandler.java

```diff
@@ -114,24 +125,66 @@ void handleRewriteObject(URI sourceResource, URI destinationResource)
    * @param project A GCP project ID where bucket created.
    * @param resource A GCS bucket URI.
    * @throws AccessDeniedException Thrown when access denied.
+   * @return Access Token.
    */
-  void handleInsertBucket(String project, URI resource) throws AccessDeniedException;
+  String handleInsertBucket(String project, URI resource) throws AccessDeniedException;
 
   /**
    * Handles <a href=https://cloud.google.com/storage/docs/json_api/v1/buckets/get>get bucket
    * request</a>.
    *
    * @param resource A GCS bucket URI.
    * @throws AccessDeniedException Thrown when access denied.
+   * @return Access Token.
    */
-  void handleGetBucket(URI resource) throws AccessDeniedException;
+  String handleGetBucket(URI resource) throws AccessDeniedException;
 
   /**
    * Handles <a href=https://cloud.google.com/storage/docs/json_api/v1/buckets/delete>delete bucket
    * request</a>.
    *
    * @param resource A GCS bucket URI.
    * @throws AccessDeniedException Thrown when access denied.
+   * @return Access Token.
    */
-  void handleDeleteBucket(URI resource) throws AccessDeniedException;
+  String handleDeleteBucket(URI resource) throws AccessDeniedException;
+
+  /**
+   * Returns the information used to acquire an access token.
```

Comment:
Is this really necessary since there's `@return` below? I'd vote to combine them together.